boards/DCM/Src/Io/Io_Imu_LSM6DS33.c

```diff
@@ -0,0 +1,189 @@
+#include "Io_Imu_LSM6DS33.h"
```

Comment:
I'd also rename this file to Io_LSM6DS33.c, and also get rid of `Imu` in the function names below

Reply:
For example `Io_Imu_LSM6DS33_writeToImu` should be changed to `Io_LSM6DS33_WriteToImu`

Reply:
changed it